src/workerd/api/global-scope.c++

```diff
@@ -477,28 +477,34 @@ void ServiceWorkerGlobalScope::sendHibernatableWebSocketMessage(
 }
 
 void ServiceWorkerGlobalScope::sendHibernatableWebSocketClose(
-    kj::String reason,
     int code,
+    kj::String reason,
+    bool wasClean,
```

Comment:
Non-blocking Nit: I know I'm fighting a losing battle on it but I'm generally not a fan of bool arguments like this. I'd much rather an enum or enum class for this kind of thing.

Reply:
In this case, I could pass a `HibernatableSocketParams::Close`, which already exists as part of this commit and looks like:

```
  struct Close {
    int closeCode;
    kj::String closeReason;
    bool wasClean;
  };
  ```

Reply:
I moved the struct to another file because there was a circular dependency, but it should be fine https://github.com/cloudflare/workerd/pull/449/commits/a602115f236beb4fb35fc8293a72080f31abd082